iohk/UpdateProposal.hs

```diff
@@ -428,6 +442,8 @@ updateProposalFindInstallers opts env = do
   res' <- moveInstallersToWorkDir opts res
   writeWikiRecord opts res'
   storeParams opts (UpdateProposalConfig2 params res')
+  echo "*** Installers are now in the work dir. Optionally sign them with:"
```

Comment:
It looks like we're signing them on line 490, so why are we telling the user to sign them manually?